TranServer/game/templates/html/home.html

```diff
@@ -2,35 +2,37 @@
 
 <!DOCTYPE html>
 <html lang="fr">
+
 <head>
   <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
 
   <link rel="stylesheet" href="{% static 'home.css' %}">
-<meta charset="UTF-8">
-<meta name="viewport" content="width=device-width, initial-scale=1.0">
-<title>Transcendance - home</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Transcendance - home</title>
 </head>
+
 <body>
   <div class="vignette"></div> <!-- Ajout de la vignette -->

```

Comment:
Insert single page container (see general chat for example) that holds everything inside of body